src/Routes.tsx

```diff
@@ -21,6 +22,9 @@ function MainRoutes() {
           <Route path="/pricing" element={<Pricing />} />
           <Route path="/pricing-form" element={<PricingForm />} />
           <Route path="/login-admin" element={<AdminLogin />} />
+          <Route path="/" element={<Perfomancetraineetable />} />
```

Comment:
- You can move this route to the dashboard routes
- Also, give it a descriptive path name for us to be able to test this.

Reply:
Done, got to URL (http://localhost:3000/dashboard/trainee/perfomance) to test it and click details there.